src/components/Modal/InfoModal/InfoModal.tsx

```diff
@@ -0,0 +1,47 @@
+import ReactDOM from "react-dom";
+import tickIcon from "../../../assets/images/tickIcon.png";
+
+import { useAppDispatch, useAppSelector } from "../../../redux/hooks";
+import {
+  selectInfoToggle,
+  selectInfoText,
+  selectInfoTitle,
+} from "../../../redux/infoModalSlice";
+import { handleInfoToggle } from "../../../redux/infoModalSlice";
+import ModalCloseButton from "../../common/Modal/ModalCloseButtons/ModalCloseButton";
+
+import infoModalStyles from "./infoModal.module.css";
+import modalStyles from "../modal.module.css";
+
+function InfoModal() {
+  const toggle = useAppSelector(selectInfoToggle);
+  const title = useAppSelector(selectInfoTitle);
+  const text = useAppSelector(selectInfoText);
+  const dispatch = useAppDispatch();
+
+  if (toggle) {
+    return ReactDOM.createPortal(
```

Comment:
Your modals are scattered all over the project

You must have one portal for the entire project, and you represent the content through children